changelog/issue-4946.md

```diff
@@ -0,0 +1,5 @@
+audience: admins
+level: patch
+reference: issue 4946
+---
+On the UI page /hooks, fix the "no hooks" detection so that hook groups are displayed.
```

Comment:
Interesting! Is this expected for all merges into tc?

Reply:
yes, see https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/changelog.md. There is a CI check that will complain if you omit the changelog. They are collected for the release notes when you release a new version.

Reply:
Appreciate the link John!